README.md

```diff
@@ -18,15 +21,20 @@ Status
 
 Nethogs is a mature piece of software included in most Linux distributions.
 
-Ideas for features, as well as open bugs, can be found at https://github.com/raboof/nethogs/issues
+Ideas for features, as well as open bugs, can be found on  [issues' label:enhancement](https://github.com/raboof/nethogs/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
```

Comment:
Yes, it's a link to the issue page filtered by _enhancement_ label. You can this the [result link in my patch](https://github.com/edouard-lopez/nethogs/blob/patch-1/README.md).